docs/pages/admin-guides/teleport-policy/database-access-controls.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Teleport Policy and Database Access Controls
+description: Describes how to import and visualize advanced Teleport Database Access Controls policies using Teleport Policy and Access Graph.
+---
+
+Advanced Database Access Controls features allow Teleport administrators to configure user permissions for individual database objects using flexible, label-based approach for expressing access policies.
+
+The database objects-level access information is automatically synchronized to Teleport Policy, making it possible to see who has particular levels of access to the different parts of the database.
+
+<Notice type="note">
+Currently PostgreSQL is the only supported protocol for database-object level permissions. Support for additional protocols will be added in the future.
+</Notice>
+
+## How it works
+
+Teleport continuously scans connected databases and imports objects, such as tables, based on predefined import rules. These rules determine which objects to import and what labels to assign to them. By default, all objects are imported, with labels automatically generated from object properties like schema and database name.
+
+The labels assigned to these objects are matched against the `spec.allow.db_permissions` section of each user role. This results in a set of permissions that will be applied when the user connects to the database.
+
+For more details of this process see [Database Access Controls](../../enroll-resources/database-access/rbac.mdx).
+
+Teleport Policy calculates the effective permissions for each user in advance, and thanks to automatic synchronization, it provides real-time insights into database access for individual Teleport users.
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v16.2.0 or later.
+- Teleport Identity and Teleport Policy enabled for your account.
+  - For self-hosted clusters, ensure that an up-to-date `license.pem` is used in the Auth Service configuration.
+- For self-hosted clusters, a running Access Graph node v1.21.3 or later.
+Check the [Teleport Policy page](./teleport-policy.mdx) for details on
+how to set up Access Graph.
+- The node running the Access Graph service must be reachable from the Teleport Auth Service.
+
+To verify that Access Graph is set up correctly for your cluster, sign in to the Teleport Web UI and navigate to the Management tab.
+If enabled, the Access Graph menu item will appear in the Permission Management section.
+
+## Example Minimal Database Access Controls Configuration
+
+This section provides an example of a Teleport configuration that corresponds to the Teleport Policy usage examples detailed later. Before deploying an updated configuration in a production environment, please ensure you've reviewed the detailed documentation on the relevant topics.
+
+### Creating a Teleport Role
```

Comment:
Isn't this just a "regular setup" for Database access control in Teleport? I think we can just link the Teleport configuration instead of repeating it here.